src/Defra.Trade.Events.DAERA.GCNotifier.Application/Defra.Trade.Events.DAERA.GCNotifier.Application.csproj

```diff
@@ -4,17 +4,13 @@
     <ImplicitUsings>enable</ImplicitUsings>
     <Nullable>enable</Nullable>
   </PropertyGroup>
+
   <ItemGroup>
-    <Compile Remove="Infrastructure\**" />
-    <EmbeddedResource Remove="Infrastructure\**" />
-    <None Remove="Infrastructure\**" />
+    <PackageReference Include="AutoMapper" Version="13.0.1" />
+    <PackageReference Include="Defra.Trade.Common.Function.Health" Version="4.0.3-pre" />
```

Comment:
Is there a full release of the nuget available?

Reply:
Yes upgraded to latest